### PR TITLE
Added advised height of the camera and display

### PR DESCRIPTION
**Pull request by MichelBouman**
Added advised height of the camera and display based on The Hive guidance.

### CODE REVIEW

File: Teams/teams-meeting-room-guidance.md [hunk 1]
```diff
@@ -325,7 +325,7 @@ A room of this type requires the use of an intelligent, ultrawide-angle front-of
 
 #### Camera placement
 
-The camera should be placed at the front of the room below the front-of-room projection but above the top of the table to capture all the meeting room participants at as close to eye level as possible.
+The camera should be placed at the front of the room below the front-of-room projection but above the top of the table to capture all the meeting room participants at as close to eye level as possible. In most scenarios the ideal height is at  3'-7" or 1090mm
```
Comment (JohanFreelancer9) [inline Teams/teams-meeting-room-guidance.md:328]:
```suggestion
The camera should be placed at the front of the room below the front-of-room projection but above the top of the table to capture all the meeting room participants at as close to eye level as possible. In most scenarios, the ideal height is 3 feet, 7 inches, or 1090mm.
```

File: Teams/teams-meeting-room-guidance.md [hunk 2]
```diff
@@ -381,6 +381,8 @@ The horizontal center line of the display must be centered on the same center li
 
 The vertical center line of the display must be no more than +/- 15 degrees offset from the eye line of a typical in-room meeting participant seated at the table.
 
+The bottom of the display should be placed at 3'8" or 1110mm.
```
Comment (JohanFreelancer9) [inline Teams/teams-meeting-room-guidance.md:384]:
```suggestion
The bottom of the display should be placed at 3 feet, 8 inches, or 1110mm. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MichelBouman, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
We don't know customers' room and furniture dimensions so best to leave guidance more generic.

**Pull request closed without merging**